src/content/docs/en/guides/images.mdx

```diff
@@ -159,6 +159,37 @@ import myImage from '../assets/my_image.png'; // Image is 1600x900
 
 <ReadMore>See details about [the `<Picture />` component properties](/en/reference/modules/astro-assets/#picture-properties) in the `astro:assets` reference.</ReadMore>
 
+### Inline `.svg` files in the HTML
+
+<p><Since v="5.0.0" /></p>
+
+To inline local `.svg` files directly in your HTML, SVG files can be imported and used like any other [Astro components](/en/basics/astro-components/#component-based-design).
```

Comment:
```suggestion
To inline local `.svg` files directly in your HTML, import the SVG file to use as an Astro component. This process assigns an `<svg>` HTML tag to a component, and requires the same conventions (e.g. capitalization) as when [using dynamic tags](/en/basics/astro-syntax/#dynamic-tags).
```
I'm assuming this is how this works under the hood, and I think the link to the explanation and caveats there is more helpful, if true!